package.json

```diff
@@ -100,8 +100,8 @@
     "@eslint-community/eslint-plugin-eslint-comments": "^4.4.1",
     "@eslint/js": "^9.15.0",
     "@fontsource/roboto": "^4.5.8",
-    "@mui/icons-material": "^6.0.1",
-    "@mui/material": "^6.0.1",
+    "@mui/icons-material": "^7.0.0",
```

Comment:
Thanks for opening this PR and looking into this! We'll need to update the peer dependency ranges above as well, not just the local devDependencies for these packages.

Reply:
I'll check the peer dependencies and make an update. Thanks!